GetConfigFiles.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python3
+import os
+import sys
+import argparse
+import json
+import pprint
+import mysql.connector
+
+
+def exec_sql(cursor, sql_stmt, params, error_msg):
+    """executes a simple sql statement"""
+    try:
+        cursor.execute(sql_stmt, params)
+    except Exception as err:
+        print(f"{str(err)} \n {error_msg}")
+        sys.exit()
```

Comment:
is this error handling useful? You are printing the error and then exiting, which is what would happen if you didn't do this at all.

Reply:
asdf

Reply:
> Why do I want to dig though the call stack when the sql statment has a simple error?

Sure, I see your point. But I guess personally I am not so bothered by a stack trace I tend to look at the last error first before going up the trace.